lib/blocks.php

```diff
@@ -48,6 +48,9 @@ function gutenberg_reregister_core_block_types() {
 				'post-tags.php'           => 'core/post-tags',
 				'site-title.php'          => 'core/site-title',
 				'template-part.php'       => 'core/template-part',
+				'query.php'               => 'core/query',
+				'query-loop.php'          => 'core/query-loop',
+				'query-pagination.php'    => 'core/query-pagination',
```

Comment:
In a single.html template i can see my self needing a similar pagination block (next/previous) but without having a wrapping query in this case since we're already on the "post" template page. Do you think there's a way to make that work?

Reply:
Yes, agreed — we discussed this briefly with examples of having pagination in the header, or a sidebar, etc.

Reply:
Yes, templates will have their query context provided from the root level so you could insert pagination under that.